### PR TITLE
wdqs-frontend: build the UI

### PR DESCRIPTION
**Pull request by maxlath**
should fix the following tickets:
* https://phabricator.wikimedia.org/T234041
* https://phabricator.wikimedia.org/T191953
* https://phabricator.wikimedia.org/T223068

### CODE REVIEW

File: wdqs-frontend/latest/default.conf [hunk 1]
```diff
@@ -14,7 +14,7 @@ server {
     }
 
     location / {
-        root   /usr/share/nginx/html;
+        root   /usr/share/nginx/html/build;
```
Comment (Ladsgroup) [inline wdqs-frontend/latest/default.conf]: I would not put it here. I will change the line 37 from:
```COPY --from=builder /tmp/wikidata-query-gui-master /usr/share/nginx/html```
To
```COPY --from=builder /tmp/wikidata-query-gui-master/build /usr/share/nginx/html````
That way we only copy the build to the docker image.

Comment (maxlath) [reply wdqs-frontend/latest/default.conf]: yes, amended commit: 83d44b2

**Comment by addshore** [comment]
This is a fairly big brekaing change for the way to image worked.
Would have been nice to have the tagged versions done before doing this, as if any users have any customizations to the image other than the basic things they will now be broken, and there is no tag to go back to.

**Comment by addshore** [comment]
For anyone that needs the prior version I looked it up and the following digest could be used:

latest: digest: sha256:9bc95c4678e1110a28ee9556598ccabcf0dbaa2be58dee2c59e8a2f71fae4b66 size: 1780

From https://travis-ci.org/wmde/wikibase-docker/jobs/590363979

Which is built from https://github.com/wmde/wikibase-docker/commit/e822265186acf18155bc1f42f7d73da15cc87536

**Pull request merged**